### PR TITLE
More whitespace fixes (Pt. 2)

### PR DESCRIPTION
**Pull request by jakirkham**
Some more whitespace fixes from @pelson and myself.

### CODE REVIEW

**Comment by jakirkham** [comment]
If I rebase this, it should actually just be one whitespace fix I think. Will go ahead and do that to make life simpler.

**Comment by pelson** [comment]
> If I rebase this, it should actually just be one whitespace fix I think. Will go ahead and do that to make life simpler.

Ah, sorry.

**Comment by pelson** [comment]
merged anyway - looks fine, even if I didn't fully understand why GH was showing such a diff.

**Comment by jakirkham** [comment]
lol. Was just about to force push. It's all good.

File: conda_smithy/templates/README.md.tmpl [hunk 1]
```diff
@@ -73,7 +73,7 @@ Current build status
 {%- set appveyor_url_name = appveyor_badge_name.replace('.', '-') %}
 
 Linux: [![Circle CI](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}}.svg?style=svg)](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}})
-OSX: [![TravisCI](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}}.svg?branch=master)](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}}) 
+OSX: [![TravisCI](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}}.svg?branch=master)](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}})
```
Comment (jakirkham) [inline conda_smithy/templates/README.md.tmpl]: This is the only real whitespace change of note that wasn't in `master`. The rest were already here.

**Pull request merged**